cmd/api/response_types.go

```diff
@@ -245,3 +258,45 @@ func NewClusterStatusChangeResponse(accountName string, clusterID string, region
 		Error:       err.Error(),
 	}
 }
+
+// NewSystemEventsListResponse creates and returns a SystemEventsListResponse instance.
+func NewSystemEventsListResponse(auditEvents []events.SystemAuditEvent) *SystemEventsListResponse {
+	numEvents := len(auditEvents)
+
+	// If there is no events, an empty array is returned instead of null
+	if numEvents == 0 {
+		auditEvents = []events.SystemAuditEvent{}
+	}
+
+	response := SystemEventsListResponse{
+		Count:  numEvents,
+		Events: auditEvents,
+	}
+	// If there is more than one event, the response contains a 'count' field
+	if numEvents > 1 {
+		response.Count = numEvents
+	}
+
+	return &response
+}
+
+// NewEventsListResponse creates and returns an EventsListResponse instance.
+func NewEventsListResponse(auditEvents []events.AuditEvent) *EventsListResponse {
+	numEvents := len(auditEvents)
+
+	// If there is no events, an empty array is returned instead of null
+	if numEvents == 0 {
+		auditEvents = []events.AuditEvent{}
```

Comment:
Not sure if this line it's needed. 
Even if the `auditEvents` array is empty, it's declared as long as you're passing arguments by value, not by reference (because it's not a pointer)